samples/bluetooth/nrf_dm/README.rst

```diff
@@ -78,6 +78,16 @@ The addresses of these devices are stored in a list.
 When a device does not respond within a certain period of time, it is removed from the list.
 Also the last measurement results with the peers are stored in this list.
 
+Zero-distance calibration
+*************************
+
+The measured values of distance measurement include an off-set.
```

Comment:
I think it is usually written "offset" without the hyphen

Reply:
Fixed.